lib/terratag-action.spec.ts

```diff
@@ -0,0 +1,73 @@
+jest.mock("axios");
+jest.mock("child_process");
+import axios from "axios";
+const mockedAxios = axios as jest.Mocked<typeof axios>;
+import childProcess from "child_process";
+const mockedChildProcess = childProcess as jest.Mocked<typeof childProcess>;
+import core from "@actions/core";
+const mockedCore = core as jest.Mocked<typeof core>;
+import tc from "@actions/tool-cache";
+const mockedTC = tc as jest.Mocked<typeof tc>;
+
+import run from "./terratag-action";
+
+test("simple end to end, latest terratag", async () => {
+  jest.resetAllMocks();
```

Comment:
Please configure jest to reset mocks for you instead of doing that on each test.  
https://jestjs.io/docs/en/configuration#resetmocks-boolean